install_config/persistent_storage/persistent_storage_glusterfs.adoc

```diff
@@ -13,567 +14,208 @@ toc::[]
-=== Creating Gluster Endpoints
+[[install-example-basic]]
+==== Example: Basic {gluster-native} Installation
 
-An endpoints definition defines the GlusterFS cluster as `*EndPoints*` and
-includes the IP addresses of your Gluster servers. The port value can be any
-numeric value within the accepted range of ports. Optionally,
-you can create a
-xref:../../architecture/core_concepts/pods_and_services.adoc#services[service]
-that persists the endpoints.
+include::install_config/persistent_storage/topics/glusterfs_example_basic.adoc[]
 
-. Define the following service:
-+
-.Gluster Service Definition
-
-[source,yaml]
-----
-apiVersion: v1
-kind: Service
-metadata:
-  name: glusterfs-cluster <1>
-spec:
-  ports:
-  - port: 1
-----
-<1> This name must be defined in the endpoints definition. If using a service, then
-the endpoints name must match the service name.
-
-
-. Save the service definition to a file, for example *_gluster-service.yaml_*,
-then create the service:
-+
-[source,bash]
-----
-$ oc create -f gluster-service.yaml
-----
+[[install-example-basic-external]]
+==== Example: Basic {gluster-external} Installation
 
+include::install_config/persistent_storage/topics/glusterfs_example_basic_external.adoc[]
 
-. Verify that the service was created:
-+
-[source,bash]
-----
-$ oc get services
-NAME                       CLUSTER_IP       EXTERNAL_IP   PORT(S)    SELECTOR        AGE
-glusterfs-cluster          172.30.205.34    <none>        1/TCP      <none>          44s
-----
+[[install-example-registry]]
+==== Example: {gluster-native} with an Integrated OpenShift Container Registry
 
+include::install_config/persistent_storage/topics/glusterfs_example_registry.adoc[]
 
-. Define the Gluster endpoints:
-+
-.Gluster Endpoints Definition
-
-[source,yaml]
-----
-apiVersion: v1
-kind: Endpoints
-metadata:
-  name: glusterfs-cluster <1>
-subsets:
-  - addresses:
-      - ip: 192.168.122.221 <2>
-    ports:
-      - port: 1
-  - addresses:
-      - ip: 192.168.122.222 <2>
-    ports:
-      - port: 1 <3>
-----
-<1> This name must match the service name from step 1.
-<2> The `*ip*` values must be the actual IP addresses of a Gluster server, not
-fully-qualified host names.
-<3> The port number is ignored.
-
-
-. Save the endpoints definition to a file, for example
-*_gluster-endpoints.yaml_*, then create the endpoints:
-+
-[source,bash]
-----
-$ oc create -f gluster-endpoints.yaml
-endpoints "glusterfs-cluster" created
-----
-
-
-. Verify that the endpoints were created:
-+
-[source,bash]
-----
-$ oc get endpoints
-NAME                ENDPOINTS                             AGE
-docker-registry     10.1.0.3:5000                         4h
-glusterfs-cluster   192.168.122.221:1,192.168.122.222:1   11s
-kubernetes          172.16.35.3:8443                      4d
-----
+[[install-example-infra]]
+==== Example: {gluster-native} for OpenShift Logging and Metrics
 
+include::install_config/persistent_storage/topics/glusterfs_example_infra.adoc[]
```

Comment:
We need to QE test this (that said I [in prior comments]) I think there are simpler ways to do this.